js/__internal/grids/grid_core/columns_controller/m_columns_controller_utils.ts

```diff
@@ -563,7 +563,12 @@ export const updateColumnChanges = function (that: ColumnsController, changeType
   }
   if (columnIndex === undefined || columnIndex !== columnChanges.columnIndex) {
     if (isDefined(columnIndex)) {
-      columnChanges.columnIndices ??= [columnChanges.columnIndex];
+      columnChanges.columnIndices ??= [];
+
+      if (isDefined(columnChanges.columnIndex)) {
+        columnChanges.push(columnChanges.columnIndex);
```

Comment:
That's my mistake. Change to this:
```suggestion
        columnChanges.columnIndices.push(columnChanges.columnIndex);
```
I guess tests have fallen because of it

Reply:
Yep, I did not notice it. Thank you.